content/docs/tasks/traffic-management/app-health-check/index.md

```diff
@@ -134,6 +134,42 @@ liveness-http-975595bb6-5b2z7c   2/2       Running   0           1m
 
 ### Mutual TLS is enabled
 
+When Mutual TLS is enabled, we have two options to support HTTP probe, liveness probe rewrites and seperate port.
+
+#### Probe Rewrite
+
+This approache rewrites the application `PodSpec` liveness probe, such that the probe request will be sent to
+sidecar. And the sidecar then redirects the request to application, and strip the response body only returning
```

Comment:
People assume sidecar == proxy
We should clarify which process in the sidecar container handles this.

Reply:
Actually I gave some thought about this. I originally was about to say `pilot agent`, however I'm not sure if that's a common concept to general istio users, or even we should expose that to them.

Reply:
https://istio.io/docs/reference/commands/pilot-agent/ is on our website. Suggest to be more accurate to avoid misleading.

Reply:
Good point. changed to pilot agent.